Update self-managed-enterprise-edition.md (#5319)
harness/developer-hub: release-notes/self-managed-enterprise-edition.md

These release notes describe recent changes to Harness Harness Self-Managed Enterprise Edition, NextGen.

:::danger Do not upgrade

> We have discovered an issue with SMP release 0.12.0 and 0.13.0 where you may face an issue with logs in UI. Please don't upgrade to 0.12.0 and 0.13.0 releases, we are working on publishing a new patch release. 

Reviewer: ```suggestion
We have discovered an issue with SMP release 0.12.0 and 0.13.0 where you might face an issue with logs in the UI. Please don't upgrade to 0.12.0 and 0.13.0 releases, we are working to publish a new patch release. 
```

> We have discovered an issue with SMP release 0.12.0 and 0.13.0, 0.13.1 and 0.13.2 where you might face an issue with logs in the UI. Please don't upgrade to 0.12.0 and 0.13.0, 0.13.1 or 0.13.2 releases, we are working to publish a new patch release and will update the patch release version here.

Reviewer: ```suggestion
We have discovered an issue with SMP releases 0.12.0, 0.13.0, 0.13.1, and 0.13.2 where you might face an issue with logs in the UI. 

Please don't upgrade to these releases.  We are working to publish a new patch release, and we will update this page with the patch release notes as soon as it's deployed.
```
